### PR TITLE
Numba parallel

### PR DESCRIPTION
**Pull request by MaxBlesch**
We want to include an example an numba parallelization.

### CODE REVIEW

File: templates/02_numba_parallel/02_numba_parallel.ipynb [hunk 1]
```diff
@@ -0,0 +1,223 @@
+{
```
Comment (peisenha) [inline templates/02_numba_parallel/02_numba_parallel.ipynb:1]: <p>Please just add a sentence or two on what this is, link to respy docs, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/ose-code-templates/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (peisenha) [inline templates/02_numba_parallel/02_numba_parallel.ipynb:1]: <p>Please use proper link usage , i.e. that people can click on <code>here</code>.  Applies to all links in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/ose-code-templates/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (MaxBlesch) [reply templates/02_numba_parallel/02_numba_parallel.ipynb:1]: <p>Done!</p>

Comment (MaxBlesch) [reply templates/02_numba_parallel/02_numba_parallel.ipynb:1]: <p>Done!</p>

**Pull request merged**